cmd/scollector/collectors/ifstat_linux.go

```diff
@@ -106,3 +112,162 @@ func c_ifstat_linux() (opentsdb.MultiDataPoint, error) {
 	})
 	return md, err
 }
+
+const (
+	linuxNetBondSlaveIsUpDesc = "The status of the bonded or teamed interface."
+	linuxNetBondSlaveCount    = "The number of slaves on the bonded or teamed interface."
+)
+
+func c_if_bond_linux() (opentsdb.MultiDataPoint, error) {
+	var md opentsdb.MultiDataPoint
+	const bondingPath = "/proc/net/bonding"
+	bondDevices, _ := ioutil.ReadDir(bondingPath)
```

Comment:
unchecked errors are bad. Especially in a case like this where the path might not exist.